hazelcast/src/main/java/com/hazelcast/spi/MemberAddressProvider.java

```diff
@@ -44,6 +45,7 @@
  * @see com.hazelcast.instance.DefaultAddressPicker
  * @see com.hazelcast.config.NetworkConfig#setMemberAddressProviderConfig(com.hazelcast.config.MemberAddressProviderConfig)
  */
+@PrivateApi
 public interface MemberAddressProvider {
```

Comment:
This is not private api, it's used by `MemberAddressProviderConfig` which is public.

Reply:
Fixed